XRTK-Core/Packages/com.xrtk.core/Extensions/TransformExtensions.cs

```diff
@@ -101,10 +101,10 @@ private static IEnumerable<Transform> EnumerateHierarchyCore(this Transform root
         /// <param name="syncTransform">
         /// True, by default, this will sync the <see cref="transform"/> rotation to calculate the axis aligned orientation.
         /// </param>
-        /// <param name="colliders">Optional cached collider collection.</param>
+        /// <param name="colliders">Referenced Collider collection that will be filled with data if none passed.</param>
         /// <returns>The total bounds of all colliders attached to this GameObject.
         /// If no colliders attached, returns a bounds of center and extents 0</returns>
-        public static Bounds GetColliderBounds(this Transform transform, bool syncTransform = true, Collider[] colliders = null)
+        public static Bounds GetColliderBounds(this Transform transform, ref Collider[] colliders, bool syncTransform = true)
```

Comment:
Documentation is missing for `syncTransform`

Reply:
it is there.